composer.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "more-cores/restcord",
```

Comment:
Shouldn't this be something like `more-cores` or `more-cores/app`? This isn't a composer file just for the restcard, no?